src/strategies.jl

```diff
@@ -124,55 +124,89 @@ end
 
 # ---------------------------------------------------------------------------------
 
+abstract StateUpdater
+
+immutable NoUpdater <: StateUpdater end
+state!(model, su::NoUpdater, obs...) = return
+
+immutable BackpropUpdater <: StateUpdater end
+function state!(model, su, target, input)
```

Comment:
yes... though doubtful that it would ever cause an actual problem.  I'll change it.